isa/Makefile

```diff
@@ -27,17 +27,32 @@ include $(src_dir)/rv32ud/Makefrag
 include $(src_dir)/rv32uzfh/Makefrag
 include $(src_dir)/rv32si/Makefrag
 include $(src_dir)/rv32mi/Makefrag
+include $(src_dir)/rv32uxpulpabs/Makefrag
+include $(src_dir)/rv32uxpulpbitop/Makefrag
+include $(src_dir)/rv32uxpulpbr/Makefrag
+include $(src_dir)/rv32uxpulpclip/Makefrag
+include $(src_dir)/rv32uxpulpmacsi/Makefrag
+include $(src_dir)/rv32uxpulpminmax/Makefrag
+include $(src_dir)/rv32uxpulppostmod/Makefrag
+include $(src_dir)/rv32uxpulpslet/Makefrag
+include $(src_dir)/rv32uxpulpvect/Makefrag
+include $(src_dir)/rv32uxpulpvectshufflepack/Makefrag
 
 default: all
 
 #--------------------------------------------------------------------
 # Build rules
 #--------------------------------------------------------------------
 
+MEMPOOL_DIR       := $(shell git rev-parse --show-toplevel 2>/dev/null || echo $$MEMPOOL_DIR)
+INSTALL_DIR       ?= $(MEMPOOL_DIR)/install
+GCC_INSTALL_DIR   ?= $(INSTALL_DIR)/riscv-gcc
+SPIKE_INSTALL_DIR ?= $(INSTALL_DIR)/riscv-isa-sim
```

Comment:
`MEMPOOL_DIR` should be more generic, not constrained to MemPool. Probably worth to `grep` and check if MemPool is mentioned somewhere else in the repo, to spot all things like these.

Also I guess that this directory definitions are tied to MemPool directory hierarchy, this would not work in any given project including riscv-tests as a submodule. To fix this we should generalize this by means of `configure`, but you can have a TO-DO note here for this.